### DISCUSSION TITLE
Feature Request: Libraries

### DISCUSSION THREAD

**Discussion by NeuroDawg** [navidrome/navidrome#3166] [Ideas]
I would love to see libraries implemented in Navidrom, just as they are done in Plex. I would like a library for pop/rock, another for jazz, another for classical, another for holiday music, etc; rather than have all that music sitting in one big library.

Yes, I could use genre tags to identify and view that music, but it's much easier to have separate libraries. Thank you for considering this feature.

**Comment by deluan** [comment]
This feature is in the roadmap: #192

**Comment by NeuroDawg** [comment]
Sorry for the duplication. I was searching in the discussions/ideas area for feature requests, not in the issues area. Is there a specific area/forum where you prefer to see feature requests?

**Reply by deluan** [reply]
No worries, you are at the right place :) The original feature request was created as an issue, way before we had this space in our GitHub page.
